wagtail/admin/templates/wagtailadmin/shared/panel.html

```diff
@@ -0,0 +1,62 @@
+{% load wagtailadmin_tags i18n %}
+{% comment %}
+    Variables this template accepts:
+
+    id_prefix - A prefix for all id attributes.
+    classes - List of CSS classes to use for the panel.
+    class_name - String of CSS classes to use for the panel.
+    id - Unique to the page.
+    heading - The text of the panel’s heading.
+    heading_size - The size of the heading.
+    icon - Displayed alongside the heading.
+    id_for_label - id of an associated field.
+    is_required - If the panel contains a required field.
+    children - Where to insert the panel’s contents.
+
+{% endcomment %}
+{% fragment as prefix %}{% if id_prefix %}{{ id_prefix }}-{% endif %}{{ id }}{% endfragment %}
+{% fragment as panel_id %}{{ prefix }}-section{% endfragment %}
+{% fragment as heading_id %}{{ prefix }}-heading{% endfragment %}
+{% fragment as content_id %}{{ prefix }}-content{% endfragment %}
+<section
+    class="w-panel {{ classes|join:' ' }}{{ class_name }}"
```

Comment:
we now have a `classnames` template tag that may help here.

```suggestion
    class="{% classnames "w-panel" classes class_name %}"
```

But, as per above, it would be really great if we did not add even more inconsistency here with these variable usage, it is very confusing for new devs and maintainers.

Reply:
Note: sorry the `classnames` template tag will not parse a list of strings but could easily do that without too many changes.